framework/test/unit/modules/random/module.spec.ts

```diff
@@ -304,8 +304,24 @@ describe('RandomModule', () => {
 			).resolves.toEqual(defaultUsedHashOnionUpdated);
 		});
 
-		it('should update the used hash onion', async () => {
+		it('should overwrite the used hash onion when forging the same height', async () => {
 			// Arrange
+			const usedHashOnionInput: UsedHashOnionStoreObject = {
+				usedHashOnions: [
+					{
+						count: 5,
+						height: 9,
+					},
+					{
+						count: 6,
+						height: 12,
+					},
+					{
+						count: 7,
+						height: 15,
+					},
+				],
+			};
```

Comment:
This object is identical to `defaultUsedHashOnionUpdated`.

I assume you didn't re-use that variable for semantic reasons...?

Reply:
I didn't modify this particular test case, `usedHashOnionInput` already existed as a duplicate. git diff is wrongly showing it as modified (picking some content from another modified test case). 

But since you pointed it out, I realized it's another duplicate test case. So, I merge it with the first case like i already did with another duplicate case.